ui/building-pages.md

```diff
@@ -0,0 +1,344 @@
+# Building Pages: Components & Widgets
+
+[[toc]]
+
+## UI Components: introduction and structure
+
+**UI components** are the basic building blocks for many UIs in openHAB 3.
+The Main UI Pages and Personal Widgets are notable UI components, but Sitemaps that were created in the UI and HABPanel dashboards are as well.
+
+These structures make up hierarchies that notably define the pages in their entirety are relatively simple:
+
+Each component has:
+
+- A **type**;
+- Most of the time, a set of **configuration properties**;
```

Comment:
I would remove those semicolons after each list element, tough i am not sure about the correct grammar.
Just feels a bit wrong to me.

Reply:
I've always done that as a habit, just checked and:

> Semicolons were traditionally used in lists to separate each bullet point, but although this is still correct, they are not used as often today. Legal writing is probably the main exception.
([source](https://www.onlinegrammar.com.au/punctuation-in-lists/)).

So I guess I'm old school, or a wannabe lawyer :)
But if they aren't used elsewhere I have no problem to remove them.